tests/manage/mcg/ui/test_verify_region_list.py

```diff
@@ -0,0 +1,120 @@
+import re
+import logging
+import time
+import pytest
+
+from ocs_ci.framework.pytest_customization.marks import tier3, polarion_id, bugzilla
+from ocs_ci.ocs.constants import NOOBAA_REGIONS_CODE_URL, AWS_REGIONS_DOC_URL
+from ocs_ci.ocs.ui.views import locate_aws_regions, locate_noobaa_regions
+from ocs_ci.ocs.ui.base_ui import BaseUI, garbage_collector_webdriver
+
+logger = logging.getLogger(__name__)
+
+
+class NavigateAWSDocsWebURL(BaseUI):
+    def __init__(self):
+        super().__init__()
+
+    def open_url(self, web_url):
+        """
+        This will open the url and returns the driver instance
+
+        Returns:
+            driver (Selenium WebDriver)
+        """
+        logger.info(f"Opening URL {web_url}")
+        self.driver.maximize_window()
```

Comment:
@mashetty330 Do you log in to anther console?

Reply:
@OdedViner Here i dont validate openshfit/odf console. this test checks official aws docs with noobaa operator code

Reply:
I didnt get what you meant by `aws-cli`? We are not testing the console. we are just comparing the supported regions in the aws official docs with the regions supported by noobaa

Reply:
If I understood correctly, we need to compare that all the zones that exist in AWS also exist in the NOOBAA code.
The question is if there is an API in which you can get all AWS regions and all NOOBAA regions, without checking the CONSOLE. 
Have you tried using CURL?

Reply:
aws-cli doesn't have option of such to list all the supported regions